src/bin/poudriere-bulk.8

```diff
@@ -111,6 +114,17 @@ arguments may be specified at once.
 .El
 .Sh OPTIONS
 .Bl -tag -width "-B name"
+.It Fl b Ar name
+Specify the
+.Ar name
+of the binary package branch to use to prefetch packages.
+Should be
+.Qq latest
+or
+.Qq quarterly .
+With this option poudriere will first try to fetch from the binary package
+repository specified the binary package prior to do the sanity check if the
```

Comment:
With this option poudriere will try to fetch binary packages from the specified repository prior to doing its sanity check on local packages.